### DISCUSSION TITLE
Benetel RAN 550 Setup Issues

### DISCUSSION THREAD

**Discussion by Higgigamer19** [srsran/srsRAN_Project#672] [General Help] [answered]
We are attempting to set up the Benetel RAN 550 and we configured exactly how the srsRAN documentation states it. We have a couple of questions regarding this issue:
1. How do we activate the AUX port on the RAN 550 as it doesn't exist as an interface on the RU?
2. Do we need DPDK for this setup?
3. How do we configure the O-RU with the DU's MAC address?

Any help would be appreciated, thank y'all.

**Comment by ffxxcad** [comment, marked as answer]
1. My understanding is that port cannot be activated. All data should go through the main interface.
2. DPDK is not necessary, but significantly helps performance.
3. Per the [Benetel User Guide](https://benetel.com/support/install-guides/), edit the /usr/sbin/radio_setup_ran550_x.sh file on the RU with the MAC address of the DU machine interface. See sections 7.2.1 and 7.2.2 for details.
